src/util.rs

```diff
@@ -92,3 +96,27 @@ mod tests {
         assert_eq!(100, limit(1000, 10, 100));
     }
 }
+
+#[cfg(not(windows))]
+pub fn start_daemon(program: &str,args: &Vec<String>) -> io::Result<std::process::Child>{
+
+    Command::new(program)
+        .args(args)
+        .before_exec(|| {
+            // Detach forked process from Alacritty. This will cause
+            // init or whatever to clean up child processes for us.
+            unsafe { ::libc::daemon(1, 0); }
+            Ok(())
+        })
+        .spawn()
+
+}
+
+#[cfg(windows)]
+pub fn start_daemon(program: &str,args: &Vec<String>) -> io::Result<std::process::Child>{
+    Command::new(program)
+        .args(args)
+        .spawn()
+
+}
+
```

Comment:
The three additional whitespaces inside the `start_daemon` methods should be removed.

Additionally a whitespace should be added before `args: &Vec<String>` and after `<std::process::Child>{`.